phpstan.neon.dist

```diff
@@ -115,7 +109,19 @@ parameters:
             paths:
                 - lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php
 
+        # This is a new requirement for PHP7's oci8 extension that prevents bound values from being garbage collected.
```

Comment:
IIRC, I checked it recently and it's not true. We can revisit it later.

Reply:
I don't know enough about Oracle and its PHP drivers to answer this question. PHPStan emitted this error, I looked up the code that caused it and found this comment there, so I just copied it over to this file.

Reply:
Indeed, this code doesn't exist in `3.2.x` as of https://github.com/doctrine/dbal/pull/4957:

> Additionally, it removes the `$boundValues` property. It might be necessary in the early PHP 7 days but doesn't seem necessary now (besides PHP 7.4 on CI, tested locally with PHP 7.3.32).

Keeping this error suppressed in `2.13.x` looks right.